modules/Expectation.js

```diff
@@ -293,77 +293,47 @@ class Expectation {
     return this
   }
 
-  toInclude(value, compareValues, message) {
+  toInclude(value, compareValues, message, _exclude = false) {
```

Comment:
Is a precedence for this already included for the `Expectation` class? My next instinct would to be to create a `TestUtils` function.